operators.js

```diff
@@ -0,0 +1,25 @@
+/* Operators
+ *See how different operators worl
+*/
+
+
+let num1 = 40; //Change Values to see how operators work
+
+let num2 = 5;
+
+console.log('value of num1 divided by num2 is: ', num1 / num2);
+console.log('Value of num1 times num2 is: ', num1*num2);
+console.log('Value of num1 plus num2 is: ',num1+num2);
+console.log('The remainder of num2 into num1 is: ',num1%num2);
+console.log(' 40 to the fifth power is: ', num1**num2);
+console.log('The rounded down value of num2 divided by num1 is: ', Math.floor(num2/num1));
+console.log('--------------------');
+
+
+
+
+//Get the right most digit of a value using '%'
+
+let val = 465; //Experiment to find which value will leave a remainder of 5. 465-460 = 5
+
+console.log("The right most digit of the value is :", val%460);
```

Comment:
@samooreSF While it's true that `val % 460` *happens* to give the right-most digit, it won't work for any number smaller than 450 or larger than 459. You want an `x` such that `val % x` is always the last digit, regardless of the value of `val`.

Reply:
For example, imagine a function `rightMostDigit(n)` that takes an integer `n` and returns the right-most digit.